### DISCUSSION TITLE
打开和关闭标签页会卡顿是什么原因呢

### DISCUSSION THREAD

**Discussion by rbetree** [ywmoyue/biliuwp-lite#495] [Q&A]
刚打开的话是正常的，看了几个视频之后，打开和关闭标签页就会有点卡顿

**Comment by ywmoyue** [comment]
每打开关闭一次视频标签页都会有一部分内存暂时没法回收，可能是这个原因

猜测是FFmpegInteropX这个库导致的，这个库最新版可能修复了内存泄漏问题但是改动有点大这边暂时还不打算升
